apis/swagger.yml

```diff
@@ -3597,6 +3597,49 @@ definitions:
           items:
             type: "string"
 
+  EventsMessage:
+    description: "represents the information an event contains"
+    type: "object"
+    properties:
+      status:
```

Comment:
Please more details about the description of all these field. It would helps a lot for others to understand these.